govuk_admin_template.gemspec

```diff
@@ -23,8 +23,6 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency "capybara", "~> 3"
   gem.add_development_dependency "govuk_test", "~> 2"
-  gem.add_development_dependency "jasmine", "~> 3"
-  gem.add_development_dependency "jasmine_selenium_runner", "~> 3"
   gem.add_development_dependency "rspec-rails", "~> 3"
   gem.add_development_dependency "rubocop-govuk"
   gem.add_development_dependency "sass-rails", "~> 5"
```

Comment:
Let me see ... we should change this because it'll complain that it's using end of life sass, we shouldn't change it if it's needed because some other dependency is ancient and needs it (perhaps bootstrap-sass).

I'd have a go at setting it to sassc-rails and see if it works?

Reply:
✅ Now using `sassc-rails`